standalone/headers-cache/Cargo.toml

```diff
@@ -12,7 +12,7 @@ clap = { version = "3", features = ["derive"] }
 tokio = { version = "1.9.0", features = ["full"] }
 chrono = { version = "0.4.19" }
 env_logger = "0.9.0"
-rocket = "0.5.0-rc.2"
+rocket = { version = "0.5.0-rc.2", git = "https://github.com/SergioBenitez/Rocket" }
```

Comment:
Headers cache doesn't touch the bug I fixed. It can still use the officially released version.

Reply:
fixed, I'm using `rocket = 0.5.0-rc.2` on Cargo.toml, and patch it on root, that will help don't compile Rocket twice